go.mod

```diff
@@ -6,7 +6,7 @@ module github.com/upbound/provider-aws
 
 go 1.23.6
 
-replace github.com/crossplane/upjet => github.com/negz/upjet v0.0.0-20250227033622-bf0ba4f0de9d
+replace github.com/crossplane/upjet => github.com/negz/upjet v0.0.0-20250320063135-520f63c242c6
```

Comment:
Just adding a note that we shouldn't merge this until we switch this to the crossplane/upjet crossplane-v2 branch after https://github.com/crossplane/upjet/pull/480 is merged.